coderd/tailnet.go

```diff
@@ -97,6 +98,18 @@ func NewServerTailnet(
 		agentConnectionTimes: map[uuid.UUID]time.Time{},
 		agentTickets:         map[uuid.UUID]map[uuid.UUID]struct{}{},
 		transport:            tailnetTransport.Clone(),
+		connsPerAgent: prometheus.NewGaugeVec(prometheus.GaugeOpts{
+			Namespace: "coder",
+			Subsystem: "servertailnet",
+			Name:      "open_conns",
+			Help:      "Total number of TCP connections currently open to workspace agents.",
+		}, []string{"agent_id"}),
```

Comment:
label values would be an unbounded set.  This would be a huge number of time series because you get a new agent ID on each build.

I don't think we can justify the value of that many.

Reply:
I didn't really take into account new builds creating new agent ids, so this makes sense :+1: